pango/src/lib.rs

```diff
@@ -56,6 +56,7 @@ pub use crate::language::Language;
 pub mod rectangle;
 pub use crate::rectangle::Rectangle;
 pub mod glyph;
+pub use glyph::{GlyphGeometry, GlyphInfo, GlyphItem, GlyphString};
```

Comment:
clippy doesn't like that so much:

```
struct import `GlyphItem` is private
struct import `GlyphString` is private
```

Otherwise seems fine to me